crates/sncast/tests/helpers/devnet.rs

```diff
@@ -34,7 +34,14 @@ fn start_devnet() {
     }
 
     Command::new("tests/utils/devnet/bin/starknet-devnet")
-        .args(["--port", &port, "--seed", &SEED.to_string()])
+        .args([
+            "--port",
+            &port,
+            "--seed",
+            &SEED.to_string(),
+            "--account-class",
+            "cairo0",
```

Comment:
Note: This is necessary, because devnet sets `cairo1` as the default from the last version. It can be removed once we support Cairo >=1 accounts in #1851